src/core/qgsmaprendererjob.cpp

```diff
@@ -350,7 +350,7 @@ void QgsMapRendererJob::cleanupJobs( LayerRenderJobs& jobs )
 QImage QgsMapRendererJob::composeImage( const QgsMapSettings& settings, const LayerRenderJobs& jobs )
 {
   QImage image( settings.outputSize(), settings.outputImageFormat() );
-  image.fill( settings.backgroundColor().rgb() );
+  image.fill( settings.backgroundColor().rgba() );
```

Comment:
By my understanding this would mean that the alpha channel of the background color is correctly respected, rather than always using a totally opaque version of the color, right?